mesos-client/examples/com/mesosphere/mesos/examples/UselessFramework.scala

```diff
@@ -1,4 +1,4 @@
-package mesosphere.mesos.examples
+package com.mesosphere.mesos.examples
```

Comment:
this needs to be move out of this example folder

Reply:
There was a long debate without a solution. Ask @alenkacz about it 😉